AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -291,6 +314,8 @@ - (void)setAsyncDelegate:(id<ASCollectionViewDelegate>)asyncDelegate
     super.delegate = (id<UICollectionViewDelegate>)_proxyDelegate;
     _asyncDelegateImplementsInsetSection = ([_asyncDelegate respondsToSelector:@selector(collectionView:layout:insetForSectionAtIndex:)] ? 1 : 0);
   }
+
+  _flowLayoutInspector.collectionView = self;
```

Comment:
Yeah, I was originally storing a reference of the collectionView in the inspector but it seems like I'm now just using this to reset the delegate property cache. Will clean this up so that it's just that.